src/chapters/jdbc-auth.xml

```diff
@@ -97,9 +97,29 @@
                         depends on the version of Java you have installed. </para>
                 </listitem>
             </varlistentry>
+            <varlistentry>
+                <term><filename>sqlserver/</filename></term>
+                <listitem>
+                    <para>Contains the SQL Server authentication extension,
+                        <filename>guacamole-auth-jdbc-sqlserver-0.9.13-incubating.jar</filename>,
+                        along with a <filename>schema/</filename> directory contains SQL Server-specific
+                        scripts requires to set up the database.  The JAR extension file will need to be
+                        placed within the <filename>GUACAMOLE_HOME/extensions</filename> folder, while the
+                        SQL Server JDBC driver must be placed within the <filename>GUACAMOLE_HOME/lib</filename>
+                        directory.</para>
+                    <para><emphasis>The SQL Server JDBC driver is not included with the extension.</emphasis>  You
+                        must obtain the JDBC driver <filename>.jar</filename> yourselfand place it in the directory.
+                        Furthermore, the SQL Server authentication extension supports a number of TDS-compatible
+                        drivers, so you must make sure the one you choose is supported by the extension, that the
+                        extension is configured properly, and that the <filename>.jar</filename> is in the correct
+                        directory.  Microsoft's JDBC driver can be downloaded from <link
+                        xlink:href="https://docs.microsoft.com/en-us/sql/connect/sql-connection-libraries#anchor-20-drivers-relational-access">
+                        this page</link>.</para>
```

Comment:
Is there a way this could be rephrased such that the link is not, simply, "this page"? I'm of the school of thought that the text of a link should represent what it's linking to.

Reply:
Fixed.